### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by FranVoz** [FortiPower/PowerFMG#10]
Hello, is this still supposed to work ? 

it seems you need to lock the adom where you want to make changes now or else you get and error saying 

"Unable to use FortiManager API (-11) No permission for the resource"

thanks.

**Comment by alagoutte** [comment]
What fortimanager release ?

**Comment by FranVoz** [comment]
FortiManager-VM64
v7.0.11 GA build0595

**Comment by alagoutte** [comment]
Do you have following the connexion procedure ? And add api json access to your user ?

https://github.com/FortiPower/PowerFMG?tab=readme-ov-file#connecting-to-the-fortimanager

**Comment by FranVoz** [comment]
Yes I sure did, Connection is OK and getting info is OK too but making changes is not, as I said it seems you need to lock the adom before doing any changes (as you have to do too if you do manual changes using the GUI)

**Comment by alagoutte** [comment]
> Yes I sure did, Connection is OK and getting info is OK too but making changes is not, as I said it seems you need to lock the adom before doing any changes (as you have to do too if you do manual changes using the GUI)

the module have limited supported feature, you need to found the API call for enable this feature

**Comment by FranVoz** [comment]
so this module commands like Remove-FMGFirewallAddress are not working on current FGM releases.

**Comment by alagoutte** [comment]
> so this module commands like Remove-FMGFirewallAddress are not working on current FGM releases.

You have an error message ?

**Comment by FranVoz** [comment]
````
Unable to use FortiManager API (-11) No permission for the resource
Au caractère C:\Program Files\WindowsPowerShell\Modules\PowerFMG\0.1.0\Private\RestMethod.ps1:184 : 13
+             throw "Unable to use FortiManager API (" + $response.resu ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Unable to use F...or the resource:String) [], RuntimeException
    + FullyQualifiedErrorId : Unable to use FortiManager API (-11) No permission for the resource
```

**Comment by FranVoz** [comment]
I think It is related to a certain adom mode that could be configured as "workflow" or "workspace" mode
I'll do some tests and come back if I find some info.
